Games/types/Mafia/roles/cards/Carol.js

```diff
@@ -24,7 +24,6 @@ module.exports = class Carol extends Card {
                         for (let action of this.game.actions[0]) {
                             if (
                                 action.actors.indexOf(this.target) != -1 &&
-                                action.priority < this.priority &&
```

Comment:
Priority should only be used for execution order.

If we want actions to not appear to tracker and watcher, just add "hidden" instead of putting them at a later priority.

Please let me know your thoughts, I might be missing some important cases here.